pkg/api/v1/types_swagger_doc.go

```diff
@@ -0,0 +1,1169 @@
+		"items":    "A list of persistent volume claims. See http://releases.k8s.io/HEAD/docs/persistent-volumes.md#persistentvolumeclaims",
+	}
+}
+
+func (PersistentVolumeClaimSpec) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":            "PersistentVolumeClaimSpec describes the common attributes of storage devices and allows a Source for provider-specific attributes",
+		"accessModes": "Contains the types of access modes required The desired access modes the volume should have. See http://releases.k8s.io/HEAD/docs/persistent-volumes.md#access-modes-1",
+		"resources":   "Resources represents the minimum resources required The desired resources the volume should have. See http://releases.k8s.io/HEAD/docs/persistent-volumes.md#resources",
+		"volumeName":  "VolumeName is the binding reference to the PersistentVolume backing this claim",
+	}
+}
+
+func (PersistentVolumeClaimStatus) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"phase":       "Phase represents the current phase of PersistentVolumeClaim",
+		"accessModes": "AccessModes contains all ways the volume backing the PVC can be mounted. The actual access modes the volume has. See http://releases.k8s.io/HEAD/docs/persistent-volumes.md#access-modes-1",
+		"capacity":    "Represents the actual resources of the underlying volume",
+	}
+}
+
+func (PersistentVolumeClaimVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"claimName": "ClaimName is the name of a PersistentVolumeClaim in the same namespace as the pod using this volume. The name of the claim in the same namespace to be mounted as a volume. See http://releases.k8s.io/HEAD/docs/persistent-volumes.md#persistentvolumeclaims",
+		"readOnly":  "Optional: Defaults to false (read/write). ReadOnly here. Will force the ReadOnly setting in VolumeMounts. Mount volume as read-only when true. Default false.",
+	}
+}
+
+func (PersistentVolumeList) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"metadata": "Standard list metadata. See http://releases.k8s.io/HEAD/docs/api-conventions.md#metadata",
+		"items":    "List of persistent volumes. See http://releases.k8s.io/HEAD/docs/persistent-volumes.md",
+	}
+}
+
+func (PersistentVolumeReclaimPolicy) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"": "PersistentVolumeReclaimPolicy describes a policy for end-of-life maintenance of persistent volumes",
+	}
+}
+
+func (PersistentVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":                     "Similar to VolumeSource but meant for the administrator who creates PVs. Exactly one of its members must be set.",
+		"gcePersistentDisk":    "GCEPersistentDisk represents a GCE Disk resource that is attached to a kubelet's host machine and then exposed to the pod. GCE disk resource provisioned by an admin. See http://releases.k8s.io/HEAD/docs/volumes.md#gcepersistentdisk",
```

Comment:
I agree a separate PR is more manageable. I think we should merge after audit, because the swagger spec is used to generate pages here http://kubernetes.io/v1.0/docs/api-reference/definitions.html. Could you open an issue for this? Thank you.